x-pack/plugins/actions/server/lib/bidirectional_connectors.ts

```diff
@@ -5,7 +5,11 @@
  * 2.0.
  */
 
-const BIDIRECTIONAL_CONNECTOR_TYPES = ['.sentinelone', '.crowdstrike'];
+const BIDIRECTIONAL_CONNECTOR_TYPES: readonly string[] = [
```

Comment:
FYI: I suspect that this will not be necessary once this merges: https://github.com/elastic/kibana/pull/203503